### PR TITLE
Add ocpMajorVersion field

### PR DESCRIPTION
**Pull request by rsevilla87**
### Description

Based on the k8s-netperf logic. Field name is different though. Shall we rename it?

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
lgtm

File: ocp-metadata/types.go [hunk 1]
```diff
@@ -83,6 +84,7 @@ type ClusterMetadata struct {
 	MetricName       string `json:"metricName,omitempty"`
 	Platform         string `json:"platform"`
 	OCPVersion       string `json:"ocpVersion"`
+	OCPMajorVersion  string `json:"ocpMajorVersion`
```
Comment (jtaleric) [inline ocp-metadata/types.go:87]: Missing quote.

**Review by jtaleric — CHANGES_REQUESTED** [review]
Missing a quote.

**Comment by vishnuchalla** [comment]
Looks good to me apart from the existing comments.

**Review by jtaleric — APPROVED** [review]

**Pull request merged**